image_tools/src/cam2image.cpp

```diff
@@ -300,6 +302,7 @@ class Cam2Image : public rclcpp::Node
   size_t width_;
   size_t height_;
   bool burger_mode_;
+  size_t device_id_;
```

Comment:
Since opencv accepts an `int` ([reference](https://docs.opencv.org/3.4/d8/dfe/classcv_1_1VideoCapture.html#a5d5f5dacb77bbebdcbfb341e3d4355c1)).
```suggestion
  int device_id_;
```